database/sqlcipher/README.md

```diff
@@ -0,0 +1,3 @@
+# sqlcipher
+
+This is just a copy of the sqlite3 driver except that it imports `github.com/mutecomm/go-sqlcipher`.
```

Comment:
link to the [sqlite3](https://github.com/golang-migrate/migrate/tree/master/database/sqlite3) db driver

Reply:
Everything done.